src/PkgAuthentication.jl

```diff
@@ -124,13 +126,12 @@ function step(state::NeedRefresh)::Union{HasNewToken, NoAuthentication, Failure}
     )
     if response isa Response && response.status == 200
         try
-            # QUESTION: should this be parsed as TOML or JSON?
             body = JSON.parse(String(take!(output)))
             if haskey(body, "token")
                 return HasNewToken(state.server, body["token"])
             end
         catch err
-            @error "invalid body received while refreshing token" exception=(err, catch_backtrace())
+            @debug "invalid body received while refreshing token" exception=(err, catch_backtrace())
```

Comment:
Are `@debug` messages printed by default? If not, this should probably be `@warn` or `@error` since we'd want the user to let us know this is happening.

Reply:
On one hand, yes, we want to know about this, but on the other hand this is "recoverable" by just requesting a new token instead of refreshing the old one.

Reply:
Fair enough. Carry on!